packages/deployment/iam/cloudformation.template.yml

```diff
@@ -481,6 +481,34 @@ Resources:
                   - s3:ListAllMyBuckets
                 Resource: '*'
 
+  ScalingRole:
```

Comment:
Why do we need a separate role for scaling?

Reply:
This is a role for DynamoDB to make changes based on current througput. Application Autoscaling (what DynamoDB uses) relies on cloudwatch alarms to trigger scaling.

This is detailed in the AWS Blog: [How to use AWS CloudFormation to configure auto scaling for Amazon DynamoDB tables and indexes](https://aws.amazon.com/blogs/database/how-to-use-aws-cloudformation-to-configure-auto-scaling-for-amazon-dynamodb-tables-and-indexes/)

and you can also see DynamoDB console requires a role in the console:
<img width="545" alt="screen shot 2018-06-28 at 6 13 54 pm" src="https://user-images.githubusercontent.com/15016780/42063383-6a888cb2-7aff-11e8-8635-8915e26fae48.png">